app/services/my_anime_list_sync_service.rb

```diff
@@ -11,36 +11,48 @@ def initialize(library_entry, method)
 
   def execute_method
     # TODO: add errors later on if we can't find mal data
-    return if mal_media.nil?
-
     case method
     when 'delete'
-      delete("#{media_type}list/#{media_type}/#{mal_media_id}", linked_profile)
+      media_type_d = library_entry['media_type'].underscore
```

Comment:
does ruby allow you to have variables names the same as method names? I wasn't sure the order it checked (whether it checked if a local variable exists, then calls method or what it does)